docs/development/contributing.rst

```diff
@@ -86,8 +86,8 @@ after this with ``git push -f``.
 
 Commit Message formatting
 -------------------------
-To ensure that all commit messages in the master branch follow a specific format, we
-enforce that all commit messages must follow the following format:
+We request that your first commit follow a particular format, and we
```

Comment:
```suggestion
We request that your first commit follows a particular format, and we
```

Reply:
I'm being very pedantic about English here, but this is a case where we should use the ["Mandative subjunctive"](https://en.wikipedia.org/wiki/English_subjunctive#Triggering_contexts).